Makefile

```diff
@@ -172,8 +172,11 @@ CLEANFILES += odh-manifests/*
 api-docs: crd-ref-docs ## Creates API docs using https://github.com/elastic/crd-ref-docs
 	$(CRD_REF_DOCS) --source-path ./ --output-path ./docs/api-overview.md --renderer markdown --config ./crd-ref-docs.config.yaml && \
 	egrep -v '\.io/[^v][^1].*)$$' ./docs/api-overview.md > temp.md && mv ./temp.md ./docs/api-overview.md
-	
+
 ##@ Build
+.PHONY: manager # to force rebuild every time since, no make dependency tracking
+manager: ## Build manager binary in the root directory rebuilding all packages and passing version
+	go build -a -ldflags "-X 'github.com/opendatahub-io/opendatahub-operator/v2/pkg/version.Version=$(VERSION)'" -o $@
 
 .PHONY: build
 build: generate fmt vet ## Build manager binary.
```

Comment:
Wouldn't it make sense to make it part of this target to? perhaps unify `go build` command for both?

Reply:
Another question, does build in Dockerfile really require -a? Doesn't it run from scratch?

Reply:
> "-a" is for doing` "go build"` in `registry.access.redhat.com/ubi8/go-toolset:$GOLANG_VERSION ` right? not the runtime from scratch. for build done in container, probably no difference to have "-a" or without it.

Yes, it is built inside of go-toolset container. So it contains some related pre-built stuff which the build rebuilds. Thanks!